base/Enums.jl

```diff
@@ -93,7 +93,7 @@ macro enum(T,syms...)
             end
         end
         function Base.show(io::IO,x::$(esc(typename)))
-            if Base.limit_output(io)
+            if get(io, :compact, false)
```

Comment:
I had been writing these as `get(io, :compact, false) === true` for future-compatibility / robustness against bad `IOContext` settings. Thoughts?